ports/stm32/boards/NUCLEO_H743ZI2/board_init.c

```diff
@@ -0,0 +1,8 @@
+#include "py/mphal.h"
```

Comment:
Could do:
```c
#include "boards/NUCLEO_H743ZI/board_init.c"
```
And then in that file, instead of using `pin_G6` use:
```
#define USB_PowerSwitchOn pyb_pin_OTG_FS_POWER
```